applications/nrf5340_audio/src/modules/lc3_playback.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _LC3_PLAYBACK_H_
+#define _LC3_PLAYBACK_H_
+
+#include <zephyr/kernel.h>
+
+/**@brief   Figure out whether or not the lc3_playback module is active
+ * @retval  true, if active. false, otherwise
```

Comment:
Since it is boolean, so there's only true, false, and no otherwise :P

Reply:
I meant "true, if active. false, if not active". Maybe this was unclear:) I can change it

Reply:
Ah... no need to change, I need to check my eyes :D